### ISSUE TITLE
Dropout in Readout Model

### ISSUE THREAD

**Issue by Tobbe02** [farizrahman4u/seq2seq#252]
Hey,
I am trying to add a Dropout-Layer to a Readout Model like this:
```python
rnn = RecurrentSequential(readout='add', teacher_force=True, return_sequences=True)
rnn.add(LSTMCell(10, input_dim=4))
rnn.add(LSTMCell(10))
rnn.add(Dropout(0.2))
rnn.add(LSTMCell(4))
``` 
but since Readout [only works with Theano](https://github.com/farizrahman4u/recurrentshop/issues/77) and the Dropout-Layer [only works with Tensorflow, ](https://github.com/keras-team/keras/blob/master/keras/layers/core.py#L80) I somehow got stuck. Does anybody have any suggestions? :)
Thanks in advance!

**Comment by Tobbe02** [comment]
pardon me. I posted this in the wrong place (too early this morning...)
